app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Please 🙏🏽 remove the foreign key here, it should be added to the user model. As I commented in the post.